dbt/adapters/teradata/connections.py

```diff
@@ -175,6 +177,26 @@ def open(cls, connection):
         # Save the transaction mode
         cls.TMODE = credentials.tmode
 
+        logger.debug("host: {}",credentials.server);
+        logger.debug("retries: {}",credentials.retries);
+
+        if credentials.retries > 0:
```

Comment:
It would be nice to have a test for this. So far, we have only done integration tests. An integration test for a retry is probably not feasible, but maybe it's time to introduce a unit test framework?

Reply:
Ok, Adam. We will plan to introduce the unit test framework.